.github/workflows/node-ci.yml

```diff
@@ -249,6 +269,8 @@ jobs:
             # Note: these two actions are locked to the latest version that were
             # published when I created this yml file (just for security).
             - name: Find existing comment
+              # Even if we're failing, we want to update the comments.
+              if: always()
```

Comment:
**note:** We want to update the PR comment if we never even published, so we always run this step.